plugins/code-coverage/package.json

```diff
@@ -33,6 +33,7 @@
     "@material-ui/lab": "4.0.0-alpha.57",
     "@material-ui/styles": "^4.11.0",
     "highlight.js": "^10.6.0",
+    "luxon": "^2.1.0",
```

Comment:
you probably want to set this to `^2.0.2` like all other packages in the project; it's usually desirable for versions to be the same across the board so we don't get double / hoisted deps

Reply:
Fixed